src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -156,7 +156,13 @@ protected ActiveEditorsOutlineService getActiveEditorsOutlineService(@NotNull Pr
    * Traverses the {@param outline} tree and adds to {@param callToTestType } the {@link DartCallExpression}s that are tests or test groups.
    */
   private void visit(@NotNull FlutterOutline outline, @NotNull Map<DartCallExpression, TestType> callToTestType, @NotNull PsiFile file) {
-    @NotNull final PsiElement element = Objects.requireNonNull(file.findElementAt(outline.getOffset()));
+    final PsiElement element = file.findElementAt(outline.getOffset());
```

Comment:
why is the outline out of sync? Aren't we applying checks to only use the outlines if the document and file have the same length? Perhaps the real fix is o make sure we are applying the conversion between editor file locations and intellij ones. On windows the lengths can be different.

Reply:
It would have stopped a couple similar bugs I had if we could make all the apis on the FlutterOutline match the IntelliJ notion of offsets rather than the Analyzer notion.